spec_creation/Validate_spec_before_upload.ipynb

```diff
@@ -241,40 +233,80 @@
     "                        lonlat_swap(wpc), popup=\"%d\" % i,\n",
     "                        icon=fof.DivIcon(class_name='leaflet-div-icon')).add_to(curr_map)\n",
     "    \n",
-    "    latlng_route_coords = [[lonlat_swap(rc) for rc in coords[\"geometry\"][\"coordinates\"]] for coords in trip[\"route_coords\"]]\n",
-    "    for ll in latlng_route_coords:\n",
-    "        folium.PolyLine(ll, popup=\"%s: %s\" % (trip[\"mode\"], trip[\"name\"])).add_to(curr_map)\n",
-    "        \n",
-    "    for i, c in enumerate(latlng_route_coords):\n",
-    "        folium.CircleMarker(c, radius=5, popup=\"%d: %s\" % (i, c)).add_to(curr_map)\n",
-    "        \n",
-    "    curr_map.fit_bounds(ful.get_bounds(trip[\"route_coords\"][0][\"geometry\"][\"coordinates\"], lonlat=True))\n",
+    "    for rc in trip[\"route_coords\"]:\n",
+    "        print(\"Found %d coordinates for the route\" % (len(rc[\"geometry\"][\"coordinates\"])))\n",
     "    \n",
+    "    latlng_routes = [[lonlat_swap(rc) for rc in r[\"geometry\"][\"coordinates\"]] for r in trip[\"route_coords\"]]\n",
+    "    for r in latlng_routes:\n",
+    "        folium.PolyLine(r, popup=\"%s: %s\" % (trip[\"mode\"], trip[\"name\"])).add_to(curr_map)\n",
```

Comment:
you need to change these as well to work with the fact the route_coords are now lists of lists, right?